src/main/java/org/sagebionetworks/web/client/widget/table/explore/StandaloneQueryWrapper.java

```diff
@@ -0,0 +1,27 @@
+package org.sagebionetworks.web.client.widget.table.explore;
+
+import org.sagebionetworks.web.client.context.SynapseContextPropsProvider;
+import org.sagebionetworks.web.client.jsinterop.QueryWrapperPlotNavProps;
+import org.sagebionetworks.web.client.jsinterop.QueryWrapperPlotNavProps.OnQueryCallback;
+import org.sagebionetworks.web.client.jsinterop.QueryWrapperPlotNavProps.OnQueryResultBundleCallback;
+import org.sagebionetworks.web.client.jsinterop.React;
+import org.sagebionetworks.web.client.jsinterop.ReactDOM;
+import org.sagebionetworks.web.client.jsinterop.SRC;
+import org.sagebionetworks.web.client.jsinterop.StandaloneQueryWrapperProps;
+import org.sagebionetworks.web.client.widget.ReactComponentDiv;
+
+public class StandaloneQueryWrapper extends ReactComponentDiv {
```

Comment:
Creates a new StandaloneQueryWrapper in a react div (which automatically cleans up on unmount)